client/signup/main.jsx

```diff
@@ -300,6 +306,13 @@ class Signup extends React.Component {
 		}
 	};
 
+	preloadNextStep() {
+		const currentStepName = this.props.stepName;
+		const nextStepName = flows.getNextStepNameInFlow( this.props.flowName, currentStepName );
+
+		nextStepName && getStepComponent( nextStepName );
```

Comment:
I wonder if it's worth `defer`ing the `getStepComponent` call here 🤔

Reply:
Good call! That way we're not blocking anything right off the bat 👌

Reply:
I wouldn't defer unless there is a serious reason to do it. `getStepComponent` just initiates a dynamic import. In webpack, that means checking if the module is already loaded or loading (lookup in the module map) and inserting a `<script>` tag with `onload` and `onerror` handlers if it isn't. That should be all pretty fast. The heavy lifting (parsing and executing the module code) is already async.

To answer an earlier question (from approx week ago): I agree that preloading the next step is a great followup to the previous work on async-loading steps. It should eliminate almost all instances where the user experiences an awkward waiting period when moving to the next step.

Reply:
> getStepComponent just initiates a dynamic import.

Oh yeah, that makes sense! and thanks for those details, I'd never gone that far through with how exactly webpack handles asyncs so this is good to know :)